source/common/json/json_streamer.h

```diff
@@ -243,7 +271,7 @@ class Streamer {
       ASSERT_THIS_IS_TOP_LEVEL;
       ASSERT(!expecting_value_);
       nextField();
-      this->streamer_.addSanitized(R"(")", key, R"(":)");
+      this->streamer_.addSanitized(Constants::Quote, key, R"(":)");
```

Comment:
See, this is your previous logic. This only replace literal quote to constexpr quote. 🤣 May it's too long time for you after you wrote this logic.

This is because we are sanitizing a map key here and the sanitized key will be quoted and an additional colon should be appended, see following example key/value pair:

`"key":` `"value"`